src/storage/txn/mod.rs

```diff
@@ -144,6 +144,9 @@ pub enum ErrorInner {
     )]
     MaxTimestampNotSynced { region_id: u64, start_ts: TimeStamp },
 
+    #[error("RawKV write fails due to potentially stale max timestamp, region_id: {region_id}")]
+    RawKvMaxTimestampNotSynced { region_id: u64 },
```

Comment:
Can MaxTimestampNotSynced be used for this purpose as well?

Reply:
The error message "Prewrite for async commit fails due to potentially stale max timestamp" of `MaxTimestampNotSynced` is not appropriate for RawKV, and I am afraid of changing the error message will break some client codes of test cases.

So I prefer to add a new error enum.